docs/solrini.md

```diff
@@ -120,9 +120,10 @@ To run end-to-end, issue the following command:
 python src/main/python/run_solr_regression.py --regression core18 --input /path/to/WashingtonPost
 ```
 
-The regression script has been verified to work for `robust04`, `core18`, and `msmarco-passage`.
+The regression script has been verified to work for [`robust04`](regressions-robust04.md), [`core18`](regressions-core18.md), and [`msmarco-passage`](experiments-msmarco-passage.md).
 
 ## Replication Log
 
 + Results replicated by [@nikhilro](https://github.com/nikhilro) on 2020-01-26 (commit [`1882d84`](https://github.com/castorini/anserini/commit/1882d84236b13cd4673d2d8fa91003438eea2d82)) for both [Washington Post](regressions-core18.md) and [Robust04](regressions-robust04.md)
 + Results replicated by [@edwinzhng](https://github.com/edwinzhng) on 2020-01-28 (commit [`a79cb62`](https://github.com/castorini/anserini/commit/a79cb62a57a059113a6c3b1523b582b89dccf0a1)) for both [Washington Post](regressions-core18.md) and [Robust04](regressions-robust04.md)
++ Results replicated by [@nikhilro](https://github.com/nikhilro) on 2020-02-12 (commit [`eff7755`](https://github.com/castorini/anserini/commit/eff7755a611bd20ee1d63ac0167f5c8f38cd3074)) for [Washington Post `core18`](regressions-core18.md), [Robust04 `robust04`](regressions-robust04.md), and [MS Marco Passage `msmarco-passage`](regressions-msmarco-passage.md) using end-to-end [`run_solr_regression`](/../../tree/master/src/main/python/run_solr_regression.py)
```

Comment:
I think a simpler relative path is something like `../src/main/python/run_solr_regression.py`?

Reply:
o nice, thanks! changed